force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -134,7 +155,19 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is in the past, False otherwise.
      */
     public static Boolean isDateInPast(Date dt) {
-        return null; // Replace null with the variable you used to store the result
+        //date that calls the method
+        Date firstDate = dt;
+        //second date
+        Date secondDate = Date.today();
+        //if date that calls the method (input) occurs after the second date (today), e.g. is in the future, the return value is negative so return false
+        Integer daysBetween =firstDate.daysBetween(secondDate);
```

Comment:
Good job searching for a way to make this work. We are going to go over date methods in week 3, so there will be an easier way to solve this.